
Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-mfdzz[0m
{"level":"warn","ts":1580147595.4627852,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580147595.4632893,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-xhkjn[0m
{"level":"warn","ts":1580147596.563083,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147596.5652714,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-lkgcj[0m
{"level":"warn","ts":1580147601.8494687,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147601.8517842,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1369-li-9hv9w[0m
{"level":"warn","ts":1580147602.1299396,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580147605.5502498,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:f3343e3a5fb3ff619430722d96ed0f53d88dc595,1369:4023572e9aa841c7988f31187068103cd4da2984
DEBUG: ran git fetch --unshallow origin 4023572e9aa841c7988f31187068103cd4da2984: f3343e3a5fb3ff619430722d96ed0f53d88dc595: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard f3343e3a5fb3ff619430722d96ed0f53d88dc595 in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 4023572e9aa841c7988f31187068103cd4da2984 in 
Merged SHA 5583590ea2f3c8debc17c82da9eaca12e71f8e4d with commit message 'Merge commit '4023572e9aa841c7988f31187068103cd4da2984'' into base branch master
Merged SHA 4023572e9aa841c7988f31187068103cd4da2984 with commit message 'can link a req to resp by id but needs more work' into base branch master
Merged SHA 8bdcd667c0451bd79e030041dfa6b6dbb668426f with commit message 'initial changes towards new request logging' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1369
+ BUILD_NUMBER=4
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Mon Jan 27 17:53:31 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1369/4.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1369 --build=4'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1369 '--comment=Mon Jan 27 17:53:31 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1369/4.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1369 --build=4'

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1369-li-42nth[0m
{"level":"warn","ts":1580147614.998955,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147615.0039675,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1369-li-js2tx[0m
{"level":"warn","ts":1580147615.2850947,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147615.5975814,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-gpvvt[0m
{"level":"warn","ts":1580147626.1817322,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580147626.1822815,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-z5dxr[0m
{"level":"warn","ts":1580147628.1319473,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147628.1340444,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-g5kkj[0m
{"level":"warn","ts":1580147634.5715535,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147634.5739527,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-wp5vs[0m
{"level":"warn","ts":1580147634.8235717,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580147636.9592397,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1369-li-lq7t5-4[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Installing collected packages: toml, attrs, appdirs, click, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
53 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
